src/draw/lv_image_dsc.h

```diff
@@ -13,21 +13,17 @@ extern "C" {
 /*********************
  *      INCLUDES
  *********************/
-#include <stdbool.h>
-#include "../misc/lv_color.h"
-#include "../misc/lv_area.h"
-#include "../stdlib/lv_string.h"
+#include "../lv_conf_internal.h"
 
 /*********************
  *      DEFINES
  *********************/
 
-#define _LV_ZOOM_INV_UPSCALE 5
-
 /** Magic number for lvgl image, 9 means lvgl version 9
  *  It must not be a valid ASCII character nor larger than 0x80. See `lv_image_src_get_type`.
  */
 #define LV_IMAGE_HEADER_MAGIC (0x19)
+LV_EXPORT_CONST_INT(LV_IMAGE_HEADER_MAGIC);
```

Comment:
As mentioned before, I suggest to rename `magic` to `signature`, as it better describes it's purpose than the generic name `magic`.
I assume this is an internal LVGL "magic number", so no API change is needed when renamed. Or do we need?